### PR TITLE
chore: added polygon and gnosis to supported networks and benchmark

### PR DESCRIPTION
**Pull request by crisgarner**
# Description

Added Polygon, Gnosis, and their testnet to supported networks, default configs, and benchmark updates.

- [x] I have read and understand MACI's [contributor guidelines](https://maci.pse.dev/docs/contributing) and [code of conduct](https://maci.pse.dev/docs/contributing/code-of-conduct).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: packages/contracts/tasks/helpers/benchmarks.ts [hunk 1]
```diff
@@ -38,9 +38,9 @@ export async function publishBatch(
   let optimalBatchSize = batchSize;
 
   while (optimalBatchSize > 0) {
+    console.log(`Attempting to publish batch of ${optimalBatchSize} messages`);
```
Comment (ctrlc03) [inline packages/contracts/tasks/helpers/benchmarks.ts:41]: was this for debugging while testing or something you think we should keep?

File: packages/contracts/tasks/helpers/benchmarks.ts [hunk 2]
```diff
@@ -38,9 +38,9 @@ export async function publishBatch(
   let optimalBatchSize = batchSize;
```
Comment (ctrlc03) [inline packages/contracts/tasks/helpers/benchmarks.ts:38]: could we add the results to https://maci.pse.dev/docs/developers-references/smart-contracts/SupportedNetworks please?

Comment (crisgarner) [reply packages/contracts/tasks/helpers/benchmarks.ts:38]: Oh, I did, I think forgot to include it in the last commit, let me upload it again

File: packages/contracts/tasks/helpers/benchmarks.ts (hunk 1 above)
Comment (crisgarner) [reply packages/contracts/tasks/helpers/benchmarks.ts:41]: I think we can remove it, I used to test when the transaction wasn't picked, but isn't really useful.

**Pull request closed without merging**